test_plans/pidgin/test_plan.md

```diff
@@ -0,0 +1,41 @@
+Pidgin is a light-weight API on which works on top of Peregrine. It takes a file's GUID as input which queries Peregrine for information about the file and returns an abstract of the file to the user.
+
+Integration Test Coverage for Pidgin :
+Pidgin has a single endpoint  - /<GUID of a file>
+
+Endpoints :	
+
+1. /<GUID of file> - this endpoint queries Peregrine and returns core metadata about the file. By default, it returns core metadata as JSON document.
```

Comment:
the 2 times you wrote `<GUID of a file>`, markdown is not displaying the text:
<img width="542" alt="Screen Shot 2019-05-31 at 9 57 43 AM" src="https://user-images.githubusercontent.com/4224001/58714617-b2342c80-838a-11e9-8bd3-088579f6b233.png">
Could you use quotes?
<img width="130" alt="Screen Shot 2019-05-31 at 10 00 27 AM" src="https://user-images.githubusercontent.com/4224001/58714728-f1627d80-838a-11e9-9bb4-5e64431d2f36.png">